src/raftstore/store/worker/compact.rs

```diff
@@ -70,11 +71,17 @@ quick_error! {
 
 pub struct Runner {
     engine: Arc<DB>,
+
+    // region_id -> last_compacted
+    raft_compacted_ctx: HashMap<u64, u64>,
 }
 
 impl Runner {
     pub fn new(engine: Arc<DB>) -> Runner {
-        Runner { engine: engine }
+        Runner {
+            engine: engine,
+            raft_compacted_ctx: HashMap::new(),
```

Comment:
no.
when the peer applied CompactLog, it saves the index to the last compacted index, and sends the previous last compacted index + current compacted index to the runner.